Userland/Libraries/LibWeb/XML/XMLDocumentBuilder.cpp

```diff
@@ -22,6 +22,9 @@ ErrorOr<DeprecatedString> resolve_xml_resource(XML::SystemID const&, Optional<XM
         return Error::from_string_literal("Refusing to load disallowed external entity");
 
     auto public_literal = public_id->public_literal;
+    if (public_literal.is_one_of("-//W3C//DTD SVG 1.1//EN")) {
```

Comment:
This bit is supposed to be in accordance with the HTML spec, we probably shouldn't even be trying to build an html document for an svg+xml file

Reply:
For reference, the list of acceptable identifiers is given in <https://html.spec.whatwg.org/multipage/xhtml.html#parsing-xhtml-documents> (though an SVG file is _not_ an XHTML file, so it shouldn't even get to this line)

Reply:
thanks for the detailed explanation 👍